code/_core/obj/item/weapon/ranged/bullet/_bullet.dm

```diff
@@ -32,7 +32,7 @@
 
 	use_iff_tag = TRUE
 
-	var/bullet_time = 0 //Not slow-mo, but how long it takes to insert a bullet into the gun.
+	var/bullet_time = list() //If there's some special bullets that take longer or shorter to load. Look at the flintlock or musket for examples
```

Comment:
This should be a null list instead an empty list, as blank lists take up memory.

``var/list/bullet_time``